index.js

```diff
@@ -7,33 +27,221 @@ function ask(questionText) {
+}
+
+const benJerrys = new Room();
+  this.roomInv = ["tables", "chairs", "VW bus", "ice cream"]
+benJerrys.message = () => {
+  console.log("Welcome to Ben & Jerrys!\nWould you like to BUY ice cream or a SHIRT?")
+}
+
+const churchSt2 = new Room();
+  this.roomInv = ["leaves", "bench"]
+churchSt2.message = () => {
+  console.log("Wow! Remember to look both ways before crossing the street.\nYou almost got smushed by a truck!")
+}
+
+const midTown = new Room();
+  this.roomInv = ["sticks", "parking meters"]
+midTown.message = () => {
+  console.log("You are standing outside 182 Main Street, home to Burlington Code Academy.\nThere is a locked door in front of you, with a 0-9 keypad on the handle.\nTry a simple 5-digit code.")
+}
+
+const acadFoyer = new Room();
+  this.roomInv = ["sign"]
+acadFoyer.message = () => {
+  console.log("There is a sign on the wall.\nIt says `BCA is on the 3rd floor. There will be coffee and tea in the kitchenette.\nIf you see the yoga studio, you went too far.`\nYou are tempted to TAKE the sign.")
+} //In-game, remember to respond if player tries to TAKE the sign ("Sorry, you can't take the sign. Stealing isn't cool, man.").
+
+const bcaLab = new Room();
+  this.roomInv = ["desks", "chairs", "laptops", "tea", "coffee", "7 Days newspaper"]
+bcaLab.message = () => {
+  console.log("Welcome to the BCA lab. Lecture will begin shortly.\nTake a COPY of 7 Days.\nHelp yourself to TEA or COFFEE, and have a seat.\n")
+} //In-game, remember to move 7 Days from roomInv to playerInv arrays. Typing TEA or COFFEE puts item in playerInv. DRINK removes beverage item from playerInv.
+
+const midTown2 = new Room();
+  this.roomInv = ["sticks", "parking meters"]
+midTown2.message = () => {
+  console.log("Whew! That was a heck of an involved lecture. You can WALK back to Church Street.")
+} //In-game, code for room change from midTown2 to churchStreet3
+
+const churchStreet3 = new Room();
+  this.roomInv = ["leaves", "bench"]
+churchStreet3.message = () => {
+  console.log("Time to go home. Nice. Would you like to WALK or RIDE the bus?")
+} //In-game, code for room change from churchStreet3 to apartment, regardless if WALK or RIDE is entered
+
+const apartMent = new Room();
+  this.roomInv = ["couch", "cat", "TV"]
+apartMent.message = () => {
+  console.log("Now you're home-sweet-home. DROP your stuff, put your feet up, and chill.\nType DONE to exit the game.")
+}
+
+function moveToRoom() {
+  room.message(); //from project instructions
+}
+
+startGame();
+
+let playerInv = ["water bottle (full)", "cigarettes", "bus fare2", "bus fare1"];
+let roomInv = [];
+
+async function startGame() {
+  let welcomeMessage = await ask (`Welcome to Zorkington!
+  This is not a game of skill. It about exploration.
+  Relax, there are no grues here. Are you ready?\n>_`)
+  if (welcomeMessage.toUpperCase === 'yes' || 'y' || 'uh huh') {
```

Comment:
remember when using logical operators you need to pass the full comparison check on each side of the operator.  Currently this always evaluates as truthy and fires no matter what you input as `'y'` and `'uh huh'` are strings and therefore always truthy.